Updated BRMS and BPM references (#348)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

*See also*: xref:zone-group[zone group]

// BxMS: Added "In Red Hat Process Automation Manager,"
[discrete]
[[realtime-decision-server]]
==== image:images/yes.png[yes] Realtime Decision Server (noun)

> *Description*: In Red Hat Process Automation Manager, the _Realtime Decision Server_ is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, JMS, or a Java client-side applications. Created as a web deployable WAR file, this server can be deployed on any web container. The current version of the Realtime Decision Server is included with default extensions for both Red Hat Process Automation Manager.

Reviewer: Can you remove "both" from the last sentence (it doesn't make sense now that there is just one listed).

*See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]

[[red-hat-process-automation-manager]]
==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)

> *Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.

Reviewer 2: ```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Bu using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies.
```
The last sentence is pretty marketing-focused, and not needed in an SSG glossary entry.

*See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]

// BxMS: General; kept as is

> [discrete]

Reviewer: @emmurphy1 can you please remove this [discrete] tag in this PR? We no longer use this in our glossary entries (see #365 for details)

[[red-hat-process-automation-manager]]
==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)

> *Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Red Hat Process Automation Manager enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and policies. It is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.

Reviewer 3: ```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). By using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
```
- Per ISG, avoid anthropomorphic phrasing such as saying that a system "enables" or "allows" humans to to do things. https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism
- `It` in the last sentence has an unclear antecedent.

*See also*:

// BxMS: Added "In Red Hat Process Automation Manager,"
[[rule]]
==== image:images/yes.png[yes] rule (noun)

> *Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic against which the rule engine to executes. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.

Reviewer 2: ```suggestion
*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic that the rule engine executes against. A rule includes a name, attributes, a “when” statement on the left side of the rule that defines the rule conditions, and a “then” statement on the right side of the rule that defines the rule results.
```

*See also*:

// BxMS: Added "In Red Hat Process Automation Manager,"
[discrete]
[[rule]]
==== image:images/yes.png[yes] rule (noun)

> *Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic for the rule engine to execute against. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.

Reviewer 3: ```suggestion
*Description*: In Red Hat Process Automation Manager, a _rule_ provides the logic against which the rule engine executes. A rule includes a name, attributes, a “when” statement on the left side of the rule, and a “then” statement on the right side of the rule.
```
- Consider rephrasing the awkward syntax in the first sentence.

*See also*:

// BxMS: Added "In Red Hat Process Automation Manager,"
[discrete]
[[runtime-manager]]
==== image:images/yes.png[yes] runtime manager (noun)

> *Description*: In Red Hat Process Automation Manager, the _runtime manager_ is an interface that enables and simplifies the usage of a KIE API within the processes. The name of the interface is `RuntimeManager`. It provides configurable strategies that control actual runtime execution. The strategies are singleton, per request, and per process instance.

Reviewer 3: ```suggestion
*Description*: In Red Hat Process Automation Manager, the _runtime manager_ interface enables and simplifies the use of a KIE API within processes. The name of the interface is `RuntimeManager`. It provides configurable strategies that control actual runtime execution. The strategies are singleton, per request, and per process instance.
```
- Consider rephrasing to avoid awkward syntax.
